spec/bolt/cli_spec.rb

```diff
@@ -593,11 +593,19 @@ def stub_config(config, file_content = nil)
       it "accepts targets resulting from --query from puppetdb" do
         cli = Bolt::CLI.new(%w[plan run foo --query nodes{}])
         allow(cli).to receive(:query_puppetdb_nodes).and_return(%w[foo bar])
```

Comment:
Lets add .once to this allowance to make sure we're not querying puppetdb multiple times in these cases.